integration_tests/jira_issues_tests.feature

```diff
@@ -119,7 +118,7 @@ Feature: Jira Issues Tests
   Scenario: D3ASIM-1690: No unmatched load
     Given we have a scenario named two_sided_pay_as_clear/jira_d3asim_1690
     And d3a is installed
-    When we run the simulation with setup file two_sided_pay_as_clear.jira_d3asim_1690 and parameters [24, 60, 60, 1]
+    When we run the d3a simulation on console with two_sided_pay_as_clear.jira_d3asim_1690 for 24 hrs (60, 60)
```

Comment:
I am a bit confused by the PR. Can you point me where is this step implementation function? Thanks

Reply:
https://github.com/gridsingularity/d3a/blob/feature/D3ASIM-2812/integration_tests/steps/integration_tests.py#L286